src/lib/types.ts

```diff
@@ -1,27 +1,44 @@
-export interface Color {
-  name: string;
-  bg: string;
-  border: string;
-}
-
 export interface User {
 	name: Username;
-	color: UserColor;
-	posX: number;
-	posY: number;
+	color: UserColorEnum;
+	cursorPosition: Coordinate;
+	drawColor: string;
 }
 
+export interface OtherUser extends User {
+	isOnline: boolean;
+}
+
+export enum UserColorEnum {
+	red, // 0
+	orange, // 1
+	yellow, // 2
+	lime, // 3
+	green, // 4
+	teal, // 5
+	brown, // 6
+	blue, // 7
+	purple, // 8
+	pink // 9
+}
+
+export type UserColor = {
+	name: string;
+	primary: string;
+	secondary: string;
+};
```

Comment:
Er der mulighed for at finde på et bedre navn end 'enum'. Det er noget som din LSP kan fortælle dig, måske den anden UserColor skal have et bedre navn, eller enum bare kan hedde color, idunno

Reply:
Jeg har bare kaldt den Color